QGIS Plugin/templateselectordialog.py

```diff
@@ -88,10 +104,13 @@ def autofit_map(self):
         map_extent = canvas.extent()
```

Comment:
Units enumerator changed in QGIS3, so `coef` variable should be now declared like this:
`coef = 1 / 0.3048 if units == QgsUnitTypes.DistanceFeet else 1`